### DISCUSSION TITLE
Docker TLS

### DISCUSSION THREAD

**Discussion by irondeau** [searxng/searxng#1119] [Q&A] [answered]
Is there a way to deploy searxng using a TLS certificate through Docker? I've been scouring the documentation and previous issues, and I haven't found anything related to this problem. Would I be able to use TLS certificates if I switched over to nginx instead?

**Comment by Erikih** [comment, marked as answer]
It is possible. There is a video tutorial on youtube on how to set it up. Keep in mind that it could take some time to generate an SSL certificate.

[https://www.youtube.com/watch?v=ifT6npY39Dw](url)

**Reply by irondeau** [reply]
I did actually end up using searxng-docker but forgot about posting this discussion question. Video was still quite informative, so thank you.
